README.md

```diff
@@ -16,7 +16,14 @@ $ go run sigs.k8s.io/kubebuilder-release-tools/notes
 $ go run sigs.k8s.io/kubebuilder-release-tools/notes -r beta
 ```
 
-## PR Verification GitHub Action
+## PR Verification GitHub Action (Deprecated)
+
+**IMPORTANT**: Images provided under `gcr.io/kubebuilder/` will be unavailable starting **March 18, 2025**. Therefore, this GitHub Action as described below will no longer work once the images are unavailable.
```

Comment:
Not sure if 18th March will be the date where it actually becomes unavailable or earlier/later.

Reply:
@sbueringer this is from the official [docs](https://cloud.google.com/artifact-registry/docs/transition/transition-from-gcr):

> After May 15, 2024, Artifact Registry will host images for the gcr.io domain in Google Cloud projects without previous Container Registry usage. After March 18, 2025, Container Registry will be shut down.

Reply:
@sbueringer yes that is the data from the docs.
Also, we confirmed it with k8s infra folks see: https://kubernetes.slack.com/archives/CCK68P2Q2/p1712626718602969?thread_ts=1712622102.206909&cid=CCK68P2Q2

In this way, that is the date that is in the Kubebuilder docs and all communications

Reply:
Okay, sounds good. I knew about the official Google communication. Didn't know that k8s folks confirmed that it will stay available until then